modules/ROOT/pages/faqs.adoc

```diff
@@ -55,114 +55,13 @@ The DataStax Luna Streaming distribution is designed for Java 11. However, becau
 === What are the install options for DataStax Luna Streaming?
 
 * Use the provided Helm Chart to install DataStax Luna Streaming in an existing Kubernetes cluster on your laptop or hosted by a cloud provider.
-* Use the provided replicated.io packaging to install on a single or multiple server(s) or VM(s).
+* Use the tarball provided at https://pulsar.apache.org/docs/en/standalone/
```

Comment:
Here we have to point to datastax/pulsar repo

See the guide here https://github.com/datastax/release-notes/blob/master/Luna_Streaming_2.8_Release_Notes.md#packaging